services/hierarchy/src/main/kotlin/IssueService.kt

```diff
@@ -81,10 +81,11 @@ class IssueService(private val db: Database) {
         )
     }
 
-    suspend fun countForOrtRunId(ortRunId: Long): Long = db.dbQuery {
+    /** Count issues found in provided ORT runs. */
+    suspend fun countForOrtRunIds(ortRunIds: List<Long>): Long = db.dbQuery {
```

Comment:
To avoid the need for callers to create lists even for single IDs, consider using `vararg ortRunIds: Long` as the parameter.

Reply:
Similar for other functions.